fe/fe-core/src/main/java/org/apache/doris/cooldown/CooldownConfHandler.java

```diff
@@ -78,7 +79,7 @@ private void updateCooldownConf(List<CooldownConf> confToUpdate) {
             if (replicas.isEmpty()) {
                 continue;
             }
-            Random rand = new Random(System.currentTimeMillis());
+            Random rand = new SecureRandom();
```

Comment:
Not need a seed any more???

Reply:
yes, securerandom can generate a more safe seed